package-kuring/Sources/Networks/Depdencies/SSEClient.swift

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright (c) 2024 쿠링
+// See the 'License.txt' file for licensing information.
+//
+
+import Foundation
+import Models
+import SwiftUI
+import Combine
+import os.log
+
+public class SSEClient: NSObject, ObservableObject, URLSessionDataDelegate {
+    @Published public var botMessage: String = ""
+    @Published public var error: Error?
+    public var sendMessage: ((String) -> Void)?
+    public static let shared = SSEClient()
+    public var task: URLSessionDataTask?
+    private var url: URL
+    private var session: URLSession?
+    private var testableFCMToken: String = Date().description
+    private let logger = Logger(subsystem: Bundle.main.bundleIdentifier!, category: "SSEClient")
```

Comment:
```suggestion
    private let logger = Logger(subsystem: "Network", category: "SSEClient")
```

굳이 force unwrapping 보다는 문자열 넣어도 될거 같아요